frontend/src/components/GddSend/TransactionConfirmationSend.vue

```diff
@@ -73,8 +77,11 @@ export default {
     email: { type: String, required: false, default: '' },
     amount: { type: Number, required: true },
     memo: { type: String, required: true },
-    loading: { type: Boolean, required: true },
-    selected: { type: String, required: true },
```

Comment:
Never used, loading is false by default and becomes true when send transaction is emitted.